src/ai/backend/manager/cli/__main__.py

```diff
@@ -7,13 +7,17 @@
 from pathlib import Path
 
 import click
+import psycopg2
 
 from ai.backend.common.cli import LazyGroup
 from ai.backend.common.logging import BraceStyleAdapter
+from ai.backend.common.validators import TimeDuration
 
 from ..config import load as load_config
 from ..models.keypair import generate_keypair as _gen_keypair
 from .context import CLIContext, init_logger
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
Please move this import after line 9, since this is an import of an external package.

Reply:
Could we replace `relativedelta` with `datetime.timedelta`?

Reply:
There are no months and years parameters in timedelta object. So In case of years, `n * 365` days is replaceable. But in case of months, it is difficult to change because it varies from month to month. What should I do?

Reply:
Hm.. What I wanted here is to unify the return type, but `relativedelta` and `timedelta` looks mostly compatible in terms of arithmetic operators and `relativedelta` has its own custom logic to handle variance of individual month sizes when calculating years and months.
Let's just leave as you first wrote. Sorry for confusion.

Reply:
Okay. Then, do I not need to update TimeDuration?

Reply:
I will merge lablup/backend.ai-common#99 soon, then please update this PR to rely on it.

Reply:
Merged it.